### DISCUSSION TITLE
Welcome to the EasyEngine community support forum 👋

### DISCUSSION THREAD

**Discussion by dhsathiya** [EasyEngine/easyengine#1502] [Show and tell]
- This forum is for user-to-user 👥 support only.
- Please select the appropriate category 💬 when starting new discussions.
- The old forum is now archived 🔒 and can be found on https://community.easyengine.io/

**Comment by fpbarros63** [comment]
Hi everyone,
I'm getting this error: [module 'typing' has no attribute '_SpecialForm'] after install processo EasyEngine in my vps ubuntu 18.04. Does anybody know why it happened?

**Reply by fpbarros63** [reply]
I repeated the commands to simulate the error but this time the messages that appear are these after running the command [wget -qO ee rt.cx/ee && sudo bash ee]:
...
Using /usr/local/lib/python3.6/dist-packages/pynginxconfig-0.3.4-py3.6.egg
Searching for psutil==3.1.1
Best match: psutil 3.1.1
Processing psutil-3.1.1-py3.6-linux-x86_64.egg
psutil 3.1.1 is already the active version in easy-install.pth

Using /usr/local/lib/python3.6/dist-packages/psutil-3.1.1-py3.6-linux-x86_64.egg
Searching for cement==2.4.0
Best match: cement 2.4.0
Processing cement-2.4.0-py3.6.egg
cement 2.4.0 is already the active version in easy-install.pth

Using /usr/local/lib/python3.6/dist-packages/cement-2.4.0-py3.6.egg
Searching for SQLAlchemy==1.4.36
Best match: SQLAlchemy 1.4.36
Processing SQLAlchemy-1.4.36-py3.6-linux-x86_64.egg
SQLAlchemy 1.4.36 is already the active version in easy-install.pth

Using /usr/local/lib/python3.6/dist-packages/SQLAlchemy-1.4.36-py3.6-linux-x86_64.egg
Searching for PyMySQL==0.8.0
Best match: PyMySQL 0.8.0
Processing PyMySQL-0.8.0-py3.6.egg
PyMySQL 0.8.0 is already the active version in easy-install.pth

Using /usr/local/lib/python3.6/dist-packages/PyMySQL-0.8.0-py3.6.egg
Searching for importlib-metadata==4.11.3
Best match: importlib-metadata 4.11.3
Processing importlib_metadata-4.11.3-py3.6.egg
importlib-metadata 4.11.3 is already the active version in easy-install.pth

Using /usr/local/lib/python3.6/dist-packages/importlib_metadata-4.11.3-py3.6.egg
Searching for greenlet==2.0.0a2
Best match: greenlet 2.0.0a2
Processing greenlet-2.0.0a2-py3.6-linux-x86_64.egg
greenlet 2.0.0a2 is already the active version in easy-install.pth

Using /usr/local/lib/python3.6/dist-packages/greenlet-2.0.0a2-py3.6-linux-x86_64.egg
Searching for zipp==3.8.0
Best match: zipp 3.8.0
Processing zipp-3.8.0-py3.6.egg
zipp 3.8.0 is already the active version in easy-install.pth

Using /usr/local/lib/python3.6/dist-packages/zipp-3.8.0-py3.6.egg
Searching for typing-extensions==4.2.0
Best match: typing-extensions 4.2.0
Processing typing_extensions-4.2.0-py3.6.egg
typing-extensions 4.2.0 is already the active version in easy-install.pth

Using /usr/local/lib/python3.6/dist-packages/typing_extensions-4.2.0-py3.6.egg
Finished processing dependencies for ee==3.8.1
.my.cnf cannot be located in your current user or root.
no crontab for root
Updating WP-CLI version to resolve compatibility issue.
**module 'typing' has no attribute '_SpecialForm'**

For EasyEngine (ee) auto completion, run the following command

source /etc/bash_completion.d/ee_auto.rc

EasyEngine (ee) installed/updated successfully
EasyEngine (ee) help: http://docs.rtcamp.com/easyengine/

**Reply by mrrobot47** [reply]
Please use the latest version
```bash
wget -qO ee rt.cx/ee4 && sudo bash ee
```
Ref: https://easyengine.io/

**Reply by fpbarros63** [reply]
> 
It worked. Thank you very much

**Comment by fpbarros63** [comment]
Continuing my journey towards learning, I'm trying to migrate my website that is hosted on hostgator to my vps to improve access speed, but when I run this command [**sudo ee site create buymania.com.br --type=wp --ssl=le**] I got the following result:

(base) fdepaula@vmi409805:~$ sudo ee site create buymania.com.br --type=wp --ssl=le
Starting site creation.
Configuring project.
Creating WordPress site buymania.com.br
Copying configuration files.
Starting site's services.
Downloading and configuring WordPress.
Moved /var/www/htdocs/wp-config.php to /var/www/wp-config.php successfully
Checking and verifying site-up status. This may take some time.

Installing WordPress site.
Success: https://buymania.com.br has been created successfully!
Starting SSL verification.
Warning: Can not validate challenge for domain buymania.com.br
Warning: Initiating clean-up.
[buymania.com.br] Docker Containers removed.
[buymania.com.br] site root removed.
Success: Site buymania.com.br deleted.
Report bugs here: https://github.com/EasyEngine/site-type-wp
(base) fdepaula@vmi409805:~$

I would really appreciate if anyone can help me clarify and solve this problem.

**Reply by marcoandrei** [reply]
@fpbarros63 create a separate issue for your question. This is the welcome message, not a support thread. Thanks.

**Comment by jauhari** [comment]
Does there any update?

**Reply by marcoandrei** [reply]
@jauhari create a separate issue for your question. This is the welcome message, not a support thread. Thanks.
